cleanlab/token_classification/filter.py

```diff
@@ -1,22 +1,24 @@
 import numpy as np
 from cleanlab.filter import find_label_issues as find_label_issues_main
-from typing import Optional, Union, Tuple
+from typing import List, Tuple
 
 
 def find_label_issues(
-    labels: list, pred_probs: list, return_indices_ranked_by: str = "self_confidence"
-) -> list:
+    labels: List[List[int]],
+    pred_probs: List[np.ndarray],  # type: ignore[type-arg]
+    return_indices_ranked_by: str = "self_confidence",
+) -> List[Tuple[int, int]]:
     """Returns issues identified by cleanlab
 
     This is a function to find the label issues for token classification datasets
 
     Parameters
     ----------
-    labels: list
+    labels: List[List[int]]
         noisy token labels in nested list format, such that `labels[i]` is a list of token labels of the i'th
         sentence. For datasets with `K` classes, each label must be in 0, 1, ..., K-1. All classes must be present.
 
-    pred_probs: list
+    pred_probs: List[np.ndarray]
```

Comment:
Redundant with type annotations, can we auto-populate these in Sphinx instead?

Reply:
Removing hard-coded tag. We'll let sphinx-autodoc-typehints handle this in a separate PR.